File: API/API_MEDICAL/Entities/Benh/Benh.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace API_MEDICAL.Entities
{
    [Table("BENH", Schema="PATIENT_RECORD")]
    public class Benh
    {
        [Key]
        [StringLength(50)]
        public string MA_BENH { get; set; }
        public string TEN_BENH { get; set; }
        [StringLength(50)]
        [ForeignKey("MA_BENH")]
        public string MA_LOAI_BENH { get; set; }
    }
}

File: App/app/src/main/java/com/example/patientrecord/Benh.java
package com.example.patientrecord;

import androidx.annotation.Nullable;

public class Benh {
    public String mA_BENH;
    public String teN_BENH;
    public String mA_LOAI_BENH;

    @Override
    public boolean equals(@Nullable Object obj) {
        if(this.mA_BENH == ((Benh)obj).mA_BENH) return true;
        else return false;
    }
}
